src/data/quality-assurance-quiz.ts

```diff
@@ -171,6 +171,17 @@ const qualityAssuranceQuiz = [
     Explanation:
       "Static Application Security Testing (SAST) involves analyzing the codebase for security vulnerabilities and violations of security rules, providing insights into potential issues before the final application build.",
     Link: "https://www.freecodecamp.org/news/4-testing-methods-which-are-mandatory-for-any-software-7731ad194fb3/"
+  },
+  {
+    Question:
+      "How many phases/stages does the Software Development Life Cycle have?",
+    Answer: "7",
+    Distractor1: "3",
+    Distractor2: "7",
```

Comment:
remember that the distractor fields represent the wrong answers only

```suggestion
    Distractor2: "12",
```